### PR TITLE
Feedbackbuttons

### PR DESCRIPTION
**Pull request by stevenernst131**
Add a button that will allow the users to email us with feedback. Also add a toast that will make people aware of this change while they are using the site.

### CODE REVIEW

File: AppLensV2/app/Common/FeedbackService.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        }
+
+        private getCurrentUrlFillMissingData(): string {
+            var currentUrl = this.$window.location.href;
+            if (currentUrl.indexOf('startTime') < 0 && currentUrl.indexOf('endTime') < 0) {
+                var startTime = 'startTime={date}'.replace('{date}', new Date(new Date().setHours(new Date().getHours() - 24)).toISOString());
+                currentUrl += (currentUrl.indexOf('?') < 0 ? '?' : '&') + startTime;
+            }
+            return currentUrl
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Common/FeedbackService.ts]: missing semicolon

**Comment by stevenernst131** [comment]
image

Opens email window:
image

It will add a start time if there is no start or end time specified

**Comment by stevenernst131** [comment]
Toast goes away after 5 seconds

File: AppLensV2/app/Main/main.html [hunk 2]
```diff
@@ -77,9 +77,13 @@ <h1 class="md-display-3">
...
             <span flex></span>
-            <md-button class="md-icon-button" aria-label="More">
-                <ng-md-icon icon="more_vert" style="fill:rgba(255,255,255,0.87)" size="30"></ng-md-icon>
+            <md-button class="md-icon-button" aria-label="Feedback" ng-click="main.sendFeedback()">
+                <ng-md-icon icon="feedback" style="fill:rgba(255,255,255,0.87)" size="30"></ng-md-icon>
             </md-button>
+            <!-- Commenting this out since it doesn't have any functionality at this point
+                <md-button class="md-icon-button" aria-label="More">
+                <ng-md-icon icon="more_vert" style="fill:rgba(255,255,255,0.87)" size="30"></ng-md-icon>
+            </md-button>-->
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Main/main.html]: You can remove this commented section. I added this earlier for feedback reason only.

**Pull request merged**